lib/packages/ui/lib/src/widgets/common/tile/user_tile.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:likeminds_chat_flutter_ui/likeminds_chat_flutter_ui.dart';
+
+class LMChatUserTile extends StatelessWidget {
```

Comment:
Should extend the `LMChatTile` class and provide customisations through its style class.

Reply:
It was redundant as we are using LMChatTile from `/src/widgets/profile/user_tile.dart`